pkg/node/address.go

```diff
@@ -423,6 +425,11 @@ func AutoComplete() error {
 
 	InitDefaultPrefix(option.Config.DirectRoutingDevice)
 
+	// in ENI mode, we do not leverage these generated CIDRs
+	if option.Config.IPAM == ipamOption.IPAMENI {
```

Comment:
Should we include Azure and Alibabacloud here as well?

Reply:
definitely, I won't be able to test **AlibabaCloud** but I updated this change to cover all 3!